addons/vehicles/config.cpp

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+
+#include "CfgVehicles.inc"
+
+class CfgPatches {
+    class ADDON {
+        name = QUOTE(COMPONENT);
+        units[] = { CLASSES_VEHICLES };
+        weapons[] = { };
+        magazines[] = { };
+        ammo[] = { };
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = { 
+            "SWS_Main",
```

Comment:
```suggestion
            "sws_main",
```

Nitpick but our addons are in lowercase. It won't break anything.

Reply:
Gotcha. I based this off of how it was in /Weapons--it looks like it's uppercase there if you want me to fix that real quick too